### ISSUE TITLE
Rewrite all examples in the `Literate` format

### ISSUE THREAD

**Issue by GTorlai** [GTorlai/PastaQ.jl#273]
- [ ] GHZ
- [ ] QFT
- [ ] XEB
- [ ] Finite temperature
- [ ] Quantum state tomography
- [ ] Quantum process tomography
- [ ] VQE
- [ ] Monitored quantum circuits

**Comment by mtfishman** [comment]
Any chance you've tried Weave?

**Comment by GTorlai** [comment]
I did not try it. But `Literate` works pretty well. Any specific reason you prefer one above the other?

**Comment by mtfishman** [comment]
No, I haven't really used either one. Whenever I've looked into this sort of thing I've come across both and didn't really understand the difference between the two. There is some discussion here: https://discourse.julialang.org/t/ann-literate-jl/10651/15

Just thought it was good to bring up the options before committing a lot of work to one or the other, in case there was some feature one had that the other didn't that we would miss out on.
